agent/src/main.cpp

```diff
@@ -107,11 +107,20 @@ int main(int argc, char **argv)
     timeoutOpt.set_arg_description("<seconds>");
     timeoutOpt.set_description("Timeout in seconds to wait for containers to shutdown, defaults to 1");
 
-    Glib::OptionEntry manifestOpt;
-    manifestOpt.set_long_name("manifest");
-    manifestOpt.set_short_name('m');
-    manifestOpt.set_arg_description("<filepath>");
-    manifestOpt.set_description("Path to a file or directory where service manifest(s) exist, defaults to nothing");
+    Glib::OptionEntry manifestDirOpt;
+    manifestDirOpt.set_long_name("manifest-dir");
+    manifestDirOpt.set_short_name('m');
+    manifestDirOpt.set_arg_description("<filepath>");
+    manifestDirOpt.set_description("Path to a file or directory where service manifest(s) exist, "
+                                   "defaults to " + std::string(SERVICE_MANIFEST_DIR_TESTING));
```

Comment:
These defines should not be here, they are explicitly exported for testing purposes. When the next config commit comes in there will be defines for these, but if they are needed now please add another definition, i.e. the some one but without "_TESTING".